### DISCUSSION TITLE
vtm-web-app broken

### DISCUSSION THREAD

**Discussion by emvazquez** [mapsforge/vtm#841] [Q&A] [answered]
The website OpenScienceMap is down and the vtm-web-app seems broken.

**Comment by devemux86** [comment]
OpenScienceMap server does not work anymore.

You need to use another tile provider in your web app.

**Reply by emvazquez** [reply]
Thank you. I will try to figure out how the project works and how to change tile providers. For the moment, I have a little trouble seeing how things fit together (vtm-web / vtm-web-js / vtm-web-app)

**Reply by schedul-xor** [reply]
For someone who wants to launch an OpenScienceMap tile server, https://github.com/schedul-xor/nextzen2oscimv4 may be another choice. It isn't completely compatible with the original server, but it seems to be working on my environment.
![image](https://user-images.githubusercontent.com/589860/137508392-ff0b3e36-efb0-47de-aa38-f8395dd3a2aa.png)

**Comment by devemux86** [comment, marked as answer]
`vtm-web` is the HTML5/GWT libGDX backend
`vtm-web-app` is the HTML5/GWT application

However `vtm-web` is not maintained very well by the community.

You could try another vector tile provider, like OpenMapTiles or Mapilion.
See their samples in [vtm-android-example](https://github.com/mapsforge/vtm/tree/master/vtm-android-example) or [vtm-playground](https://github.com/mapsforge/vtm/tree/master/vtm-playground).
It is unknown if and how Mapbox decoder can work with `vtm-web` and GWT.
